mirage/config.ml

```diff
@@ -106,7 +106,7 @@ let packages =
     package ~build:true
       ~pin:
         "git+https://github.com/TheLortex/ocaml-yaml.git#7e1f117645ea10fbec2bd3dbbf0d8f581cce891f"
-      "yaml";
+      ~pin_version:"3.0.0" "yaml";
```

Comment:
I'd propose to use `3.0.0.dev` or `3.0.0-dev` or `3.0.0~dev` to indicate for later that this wasn't the bare 3.0.0 version.

Reply:
Which one ? :) mirage uses `~` for the beta versions so I did that.

Reply:
Depends on the actual constraints, since they're ordered below and above 3.0.0 ;)

mirage-runtime.4.0 < mirage-runtime.4.0.0~beta < mirage-runtime.4.0.0